aces/Loris
# [jsx] Add required to SearchableDropdown input element

addresses https://redmine.cbrain.mcgill.ca/issues/14821

this PR adds a front-end validation, makes it so that you are unable to "save" a form if a required searchable dropdown is empty

to test on Media Upload:
1. on branch 20.0-release: Click 'Upload File' with all the fields empty. The form will prompt you to fill the required fields before the submission is able to go through. Fill in the required fields field by field and notice that you are never prompted to fill in Site.
2. Do this again on by branch, and the field for Site now prompts you to fill it in
